pkg/vz/pipe.go

```diff
@@ -0,0 +1,107 @@
+//go:build darwin && !no_vz
+// +build darwin,!no_vz
+
+package vz
+
+import (
+	"context"
+	"encoding/binary"
+	"io"
+	"net"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"inet.af/tcpproxy"
+)
+
+// PipeToQemuConn support connecting to QEMU supported network stack via unix socket
+// Returns os.File, connected dgram connection to be used for vz
+func PipeToQemuConn(unixSock string) (*os.File, error) {
```

Comment:
Doesn't seem a "pipe".

Maybe we can just call it `DialQemu`

Reply:
Done